From 811d6e81f90a5bb5697bceb9d70df60041685d62 Mon Sep 17 00:00:00 2001
From: Thao Luong <thao.luong.uw@rvc.renesas.com>
Date: Wed, 27 May 2015 10:33:09 +0700
Subject: [PATCH 02/25] gstomx: set the default path of gstomx.conf

This prioritizes the path obtained from GST_OMX_CONFIG_DIR
environment variable in the system.

refs #69206
Signed-off-by: Thao Luong <thao.luong.uw@rvc.renesas.com>
---
 omx/gstomx.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/omx/gstomx.c b/omx/gstomx.c
index faa4c4a..373ecb2 100644
--- a/omx/gstomx.c
+++ b/omx/gstomx.c
@@ -2644,6 +2644,9 @@ plugin_init (GstPlugin * plugin)
   GST_DEBUG_CATEGORY_INIT (gst_omx_video_debug_category, "omxvideo", 0,
       "gst-omx-video");
 
+  /* Set the default path of gstomx.conf */
+  g_setenv (*env_config_name, "/etc", FALSE);
+
   /* Read configuration file gstomx.conf from the preferred
    * configuration directories */
   env_config_dir = g_strdup (g_getenv (*env_config_name));
-- 
2.4.0

